pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -573,7 +602,7 @@ private static int trueCount(final Boolean... b) {
     }
   }
 
-  private File getNodePrivateKeyFile() {
+  private File getNodePrivateKeyFile(@Nullable final File nodePrivateKeyFile) {
```

Comment:
nit: Omit the `@Nullable`

Reply:
this is intended to prevent thinking the nodePrivateKeyFile value is always null as it's the initial value and no other is assigned in code but only by PicoCLI at runtime by reflexion. How should I indicate that another way ? perhaps a check disabling annotation but It would be IDE specific I guess...